Packs/CommonScripts/Scripts/ExtractHyperlinksFromOfficeFiles/ExtractHyperlinksFromOfficeFiles.py

```diff
@@ -34,10 +35,12 @@ def extract_hyperlinks_from_xlsx(file_path: str) -> Set:
 def extract_hyperlinks_from_docx(file_path: str) -> Set:
     doc = Document(file_path)
     links = set()
-    for para in doc.paragraphs:
-        for hyper in para.hyperlinks:
-            if hyper.address:
-                links.add(hyper.address)
+    rels = doc.part.rels
+    for rel in rels:
+        rel = rels[rel]
```

Comment:
```suggestion
    for rel in doc.part.rels.values():
```
assuming it's a dictionary, based on `tcpiper`'s comment [here](https://stackoverflow.com/questions/36235031/where-is-the-word-rels-document-xml-rels-in-python-docx-object)